src/main/kotlin/com/chattriggers/ctjs/minecraft/objects/KeyBind.kt

```diff
@@ -16,19 +16,20 @@ class KeyBind {
     /**
      * Creates a new key bind, editable in the user's controls.
      *
+     * @param category the keybind category the keybind will be in
      * @param description what the key bind does
      * @param keyCode     the keycode which the key bind will respond to, see Keyboard below. Ex. Keyboard.KEY_A
      * @see [Keyboard](http://legacy.lwjgl.org/javadoc/org/lwjgl/input/Keyboard.html)
      */
-    constructor(description: String, keyCode: Int) {
+    constructor(category: String, description: String, keyCode: Int) {

```

Comment:
For the three methods you modify, make the category default to "ChatTriggers" and add a `@JvmOverloads` tag so that this change is only additive, and people who don't want to specify the category don't have to.